test/unit/modules/api-translateSpec.js

```diff
@@ -36,11 +36,15 @@ describe('Unit: Testing api-translate', function () {
     describe('when translating with the service...', function () {
 
         it('should return the translated string if key exists', inject(function ($translate) {
-            $translate('SALUTE').should.be.equal('Hello');
+            $translate('SALUTE').then(function(translation) {
+                translation.should.be.equal('Hello');
+            })
         }));
 
         it('should return the original string if key does not exist', inject(function ($translate) {
-            $translate('not translated').should.be.equal('not translated');
+            $translate('not translated').then(function (translation) {
+                translation.should.be.equal('not translated');
```

Comment:
Please change test to async. This assertion does not fail the test if wrong. Also check previous test.

Reply:
Hi @aecz  I changed the test to async, and also added sync tests using the sync method `$translate.instant(`. Is this ok for you? Thanks!